backend/modules/simulation/services/grading_service.py

```
@@ -0,0 +1,216 @@
+                # Return existing grading from instance_data
+                return {
+                    "overall_score": grading_data.get("ai_grade"),
+                    "overall_feedback": grading_data.get("overall_feedback", ""),
+                    "scenes": grading_data.get("scenes", []),
+                    "rubric_total_points": grading_data.get("rubric_total_points", 100)
+                }
+        
+        # Import grading agent (lazy import to avoid circular dependencies)
+        try:
+            from modules.simulation.agents.grading_agent import grading_agent
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)